INPTPZ1/Fractals/NewtonFractals.cs

```diff
@@ -0,0 +1,94 @@
+﻿using INPTPZ1.Arguments;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+
+namespace INPTPZ1.Fractals
+{
+    public class NewtonFractals
+    {
+        private static readonly int IterationLimit = 30;
+        private static readonly double NewtonsFractalLimit = 0.5;
+        private static readonly int RGBColorValue = 255;
+        private static readonly Color[] colors =
+        {
+            Color.Red, Color.Blue, Color.Green,
+            Color.Yellow, Color.Orange, Color.Fuchsia,
+            Color.Gold, Color.Cyan, Color.Magenta
+        };
+        private static Polynomial StartFunction()
```

Comment:
methods should have a verb in their name - also repeats for many other methods below this